_api-reference/profile.md

```diff
@@ -756,274 +995,11 @@ Field | Description
 `build_aggregation`| Contains the time spent running the aggregation’s `buildAggregations()` method, which builds the results of this aggregation.
```

Comment:
Add the below sentence to the end.
```
For concurrent segment search,`build_aggregation` method contains the total elapsed time across all slices (the difference between the last completed slice execution end time and the first slice execution start time).
```

Reply:
Same information above should add it to `initialize`, `build_leaf_collector`, `collect`, and `post_collection` methods.